pkg/services/featuremgmt/registry.go

```diff
@@ -222,10 +222,11 @@ var (
 			Created:           time.Date(2022, time.September, 27, 12, 0, 0, 0, time.UTC),
 		},
 		{
-			Name:            "entityStore",
-			Description:     "SQL-based entity store (requires storage flag also)",
+			Name:            "unifiedStorage",
```

Comment:
@DanCech -- i renamed this while we are here futzing with feature flags.  In general we will want to clear all reference to "entity" but after merging :)